treeherder/model/models.py

```diff
@@ -189,7 +189,9 @@ class Bugscache(models.Model):
     crash_signature = models.TextField(blank=True)
     keywords = models.TextField(blank=True)
     os = models.CharField(max_length=64, blank=True)
-    modified = models.DateTimeField()
+    modified = models.DateTimeField(null=True)
+    created = models.DateTimeField(db_index=True, null=True)
```

Comment:
The created date field will be used for the intermittent's view endpoints (searching by date the bug was created versus modified, per your previous suggestion).

Reply:
Ah the legacy OrangeFactor site's date is "date of occurrence of this instance of an intermittent" which I think should be either the time the job started or the time the job finished, rather than the creation time of the bug as filed in Bugzilla

Reply:
oh, I see what you mean. Looking at the classification_mirroring and OF server 'date' is job.submit_time and job.start_time is used in the bug details table. But, does it make sense to search for intermittent's failures based on the associated job submit times?

Reply:
Actually just re-thought about this. The point is to find all the jobs associated with bugs that are failing during a specified time period. :)